packages/core/primitives/event-dispatch/src/event.ts

```diff
@@ -64,9 +67,14 @@ export function addEventListener(
   if (isCaptureEventType(eventType)) {
     capture = true;
   }
-  element.addEventListener(eventType, handler, capture);
 
-  return {eventType, handler, capture};
+  if (typeof passive === 'boolean') {
+    element.addEventListener(eventType, handler, {capture, passive});
+  } else {
+    element.addEventListener(eventType, handler, capture);
+  }
```

Comment:
We can probably save some bytes (and similarly for the removeEventListener fn below):
```suggestion
  const options = typeof passive === 'boolean' ? {capture, passive} : capture;
  element.addEventListener(eventType, handler, options);
```

Reply:
Done.